internal/http/actions.go

```diff
@@ -65,12 +79,20 @@ func DeletePosts(ctx *gin.Context) {
 
 func GetPosts(ctx *gin.Context) {
 	param := ctx.Param("id")
-	id, err := uuid.Parse(param)
```

Comment:
why did you remove the uuid type support? Don't you think that we can avoid reaching out the database if the user is sending a non uuid ? (eg.: `GET /posts/filhodanuvem`

Reply:
realmente, não pensei nisso